streamparse/cli/submit.py

```diff
@@ -161,7 +162,7 @@ def submit_topology(name=None, env_name=None, options=None, force=False,
     use_venv = env_config.get('use_virtualenv', True)
 
     # Check if user wants to install virtualenv during the process
-    install_venv = env_config.get('install_virtualenv', use_venv)
+    install_venv = env_config.get('install_virtualenv', not skip_virtualenv)
```

Comment:
A change that would be more consistent with our recent changes would be to change this line (and the `use_venv` line above it) to read

```py
use_venv = options.get('use_virtualenv', True)
install_venv = options.get('install_virtualenv', use_venv)
```

And put that after we call `resolve_options`.

That would let people specify `sparse submit -o install_virtualenv=False` to skip that installation without needing to create a new setting.

Reply:
@dan-blanchard Thanks for the excellent suggestion, this is way cleaner!
I've reverted my changes and implemented that in https://github.com/Parsely/streamparse/pull/421/commits/0c331168b92f656bcf4851789a9e326c8b7a1e40, let me know if that looks good to you.